identifiers/country-cl/regions.csv

```diff
@@ -1,17 +1,17 @@
 id,name
-ocd-division/country:cl,Chile
-ocd-division/country:cl/region:ai,Aisén del General Carlos Ibañez del Campo
-ocd-division/country:cl/region:an,Antofagasta
-ocd-division/country:cl/region:ar,Araucanía
-ocd-division/country:cl/region:ap,Arica y Parinacota
-ocd-division/country:cl/region:at,Atacama
-ocd-division/country:cl/region:bi,Bío-Bío
-ocd-division/country:cl/region:co,Coquimbo
-ocd-division/country:cl/region:li,Libertador General Bernardo O'Higgins
-ocd-division/country:cl/region:ll,Los Lagos
-ocd-division/country:cl/region:lr,Los Ríos
-ocd-division/country:cl/region:ma,Magallanes
-ocd-division/country:cl/region:ml,Maule
+ocd-division/country:cl/region:ai,Región de Aysén del General Carlos Ibáñez del Campo
```

Comment:
We don't usually to "Región de". For example, we don't do "State of California", "Province of Ontario", etc. The type information is given in the `region:` prefix.